ProcessManager.py

```diff
@@ -25,12 +35,17 @@ def isEmpty(self):
         else:
             return False
 
-    def createProcess(self):
+
+    def createProcess(self, name, command):
         """
         TODO create a function that takes in parameters and uses those parameters to
         ceate one single process. The process must then be stored in "self.T".
         """
 
+        process_list[name] = RoboProcesses(command)
+        return process_list
```

Comment:
Once `process_list` is an atribute of the ProcessManager, it won't need to be returned here. We could have it so `createProcess` returns true or false if a failure occured.